lib/Models/TimeSeriesStack.js

```diff
@@ -0,0 +1,73 @@
+"use strict";
+
+/*global require*/
+
+var knockout = require('terriajs-cesium/Source/ThirdParty/knockout');
+var defined = require('terriajs-cesium/Source/Core/defined');
+
+/**
+ * Manages a stack of all the time series layers currently being shown and makes sure the clock provided is always tracking
+ * the highest one. When the top-most layer is disabled, the clock will track the next highest in the stack. Provides access
+ * to the current top layer so that can be displayed to the user.
+ *
+ * @param clock The clock that should track the highest layer.
+ * @constructor
+ */
+var TimeSeriesStack = function(clock) {
+    this.clock = clock;
+
+    this._layerStack = [];
+
+    knockout.track(this);
+
+    /**
+     * The highest time-series layer, or undefined if there are no time series layers.
+     */
+    knockout.defineProperty(this, 'topLayer', {
+        get: function () {
+            if (this._layerStack.length) {
+                return this._layerStack[this._layerStack.length - 1];
```

Comment:
You probably want `_layerStack` in the `knockout.track` above.  Otherwise, I can't see much point in `topLayer` being a knockout property, or in subscribing to it, because it will never change after it's computed once.